lightkurve/correctors.py

```diff
@@ -17,11 +17,15 @@
 from astropy.stats import sigma_clip
 
 from .utils import channel_to_module_output
-from .lightcurve import LightCurve
-from .lightcurvefile import KeplerLightCurveFile
+from .lightcurve import LightCurve, KeplerLightCurve, TessLightCurve
+from .lightcurvefile import KeplerLightCurveFile, TessLightCurveFile
+from .targetpixelfile import KeplerTargetPixelFile, TessTargetPixelFile
 
+from sklearn.decomposition import PCA
```

Comment:
Is this better than what we use for the CBV corrector? Should we consider updating the CBV corrector in a future PR?

Reply:
PCA from `scikit-learn` is what I first learned, and I'm not sure how it compares to what is already used in the `CBVCorrector`. If you have thoughts about how to do a head-to-head comparison we could give it a try!

Reply:
`CBVCorrector` doesn't actually carry out PCA, it merely reads in basis vectors from the pipeline-produced CBV files.  Adding an extra dependency (sklearn) comes at some cost, but it doesn't look like numpy or scipy have an easy-to-use PCA tool, so I think I'm fine with this!